Voycar.Api.Web/Features/Members/Post/Registration/Mapper.cs

```diff
@@ -0,0 +1,26 @@
+namespace Voycar.Api.Web.Features.Members.Post.Registration;
+
+using System.Security.Cryptography;
+using Entities;
+
+public class Mapper : Mapper<Request, Response, Member>
+{
+
+    public override Member ToEntity(Request r) => new()
+    {
+        Email = r.Email,
+        PasswordHash = BCrypt.Net.BCrypt.HashPassword(r.Password),
```

Comment:
Man könnte argumentieren `BCrypt.EnhancedHashPassword()` und entsprechend `EnhancedVerify()` zu verwenden, um längere Passwörter zu unterstützen (siehe Projekt [README](https://github.com/BcryptNet/bcrypt.net?tab=readme-ov-file#enhanced-entropy). Aber das ist, in diesem Kontext, glaube ich nicht notwendig.

Reply:
> Aber das ist, in diesem Kontext, glaube ich nicht notwendig.

Ich stimme zu, passt erstmal so denke ich.